### ISSUE TITLE
Ability to make selected addons always show fist in the popup.

### ISSUE THREAD

**Issue by mxmou** [ScratchAddons/ScratchAddons#1250]
**Is your feature request related to a problem? Please describe.**

There are some addons (2.0 → 3.0, dark mode) that I want to be able to easily enable or disable, which can be harder if many addons are running on the current page so the list gets long.

**Describe the solution you'd like**

Ability to "pin" addons to the top of the addons tab.

**Comment by WorldLanguages** [comment]
I suppose this is for testing - one workaround you can use is rename the addon to "000 Website dark mode" so that it stays on top. If the addon has no userscripts, make sure to add one so that it does not fall to the "not running addons" list.

**Comment by mxmou** [comment]
> I suppose this is for testing - one workaround you can use is rename the addon to "000 Website dark mode" so that it stays on top. If the addon has no userscripts, make sure to add one so that it does not fall to the "not running addons" list.

It would also be useful for testing, but I often want to take a screenshot for the forums and I don't want it to be completely different from normal Scratch. I use the stable version for everyday use, so I don't have the ability to rename addons.

**Comment by WorldLanguages** [comment]
I see, I'm not sure if "pinning" is the best way to achieve this tho. You basically want to disable all themes at once, I guess.

**Comment by mxmou** [comment]
@WorldLanguages You're right, a checkbox to temporarily disable all addons from the popup would be useful too.

**Comment by TheColaber** [comment]
My suggestion for this issue:
- add an option in more settings/on right click of extension icon to temporarily disable all themes
- if pinning would also be useful for development, which it would, add a new addon manifest property such as `mode: "dev"` which puts the addon at the top of the "currently running on page list" and all the other lists.

**Comment by mxmou** [comment]
Someone suggested this in the comments on the v1.40 update video:
![image](https://github.com/user-attachments/assets/dc5d3dc2-f168-4b09-a4da-adeac16f8974)
